### PR TITLE
filter bot requests poc

### PR DESCRIPTION
**Pull request by martenkive**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -79,5 +128,23 @@ func main() {
...
+			if err != nil {
+  				fmt.Println(err)
+			}
+			message := fmt.Sprintf("Ignored request with userAgent %s and full request %s", userAgent, string(requestDump))
+			log.Println(message)
+			w.WriteHeader(http.StatusOK)
+			return
+            
+        } else {
+			proxy.ServeHTTP(w, r)
```
Comment (Glennrs) [inline main.go:146]: Should the else part just fall through to http.ListenAndServe like it did before?

**Review by Glennrs — COMMENTED** [review]
Not sure if we need the else logic for when it isn't a bot?

File: main.go (hunk 1 above)
Comment (martenkive) [reply main.go:146]: maybe :) Not super familiar with these functions - but the current approach seems to work. If I put the `ListenAndServe` from row 149 inside the else it doenst :(

Comment (Glennrs) [reply main.go:146]: I was thinking if you remove the else entirely and just fall through to the old code?

**Pull request merged**